### ISSUE TITLE
Q&A: Defect: Questions in list (in modal) no longer open the Q&A in another tab

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#14604]
### Expected behavior

When adding a reusable Q&A to a Q&A Group, a modal is opened with a list of Q&As to select. The titles are hyperlinked, and clicking one opens the selected Q&A in a new tab (per original [implementation ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13220)).

### Observed behavior

Steps to reproduce:
1. Go to the Edit screen for a Resources & Support node on Staging (e.g. https://staging.cms.va.gov/node/53778/edit)
2. Create/expand a Q&A Group content block
3. Click "Place Q&As" button to open modal
4. Click a Q&A title in the modal
5. See that it opens the Single Q&A node within the modal frame

Screenshot:
- Notice that the modal title is "Place Q&As" but the content within the modal is a single Q&A's actual content.
- 
<img width="802" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/4054752/f377fdb6-857b-4cc2-94ec-2d8a86fd207b">

## Acceptance criteria

- [x] Researched why this regressed from #13220
- [x] Fixed it so that clicking on a Q&A title opens the Q&A in a new tab

**Comment by chri5tia** [comment]
The config is still in place. Still looking into why it's not working.
<img width="312" alt="Screenshot 2023-08-04 at 4 04 49 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/40211890-9837-4784-9c51-e3bf8e54ac38">

config/sync/core.entity_form_display.paragraph.q_a_group.default.yml

**Comment by chri5tia** [comment]
Link opens in a new tab if you go to a node with an existing placed single Q&A in edit mode:
<img width="610" alt="Screenshot 2023-08-07 at 4 07 39 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/4b873be0-5552-4ae0-adce-88617dfd3c36">

Described in va_entity_browser custom module linked tittle display widget.

It does *not* open in a separate window when viewing from the modal during placement or on the view node page.
<img width="526" alt="Screenshot 2023-08-07 at 4 13 06 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/6b4d831d-7a22-4db6-b601-4003197522a9">

<img width="526" alt="Screenshot 2023-08-07 at 4 13 06 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/22764938/ca692bcc-ca19-406c-be09-4de157ceb9f7">

**Comment by chri5tia** [comment]
@dsasser I feel like this is something I didn't check for during code review, can you confirm if it feels accurate? I want to rule out that previous work done on this didn't get somehow reverted. e.g. Do you know if the link opened up in a new window in the modal and not just as placed on the edit page?

## Steps to reproduce my findings

1. Go to https://staging.cms.va.gov/node/51915/edit
2. Click on "edit" at "Main content" > "Q&A group Why should I create an account to sign in to VA.gov?, What ..."
3. Click on "Why should I create an account to sign in to VA.gov?" under Place Q&As > Title
The content opens in a new window as it should.

4. Now click on "Place Q&As" to add a new one.
5. Now click on the link "How do I create an ID.me account?" Or any of the links in the modal.
The link opens up inside the modal.

**Comment by dsasser** [comment]
@wesrowe @chri5tia @jilladams 

tl;dr: The isn't an identifiable regression here but we can make the links the modal open in a new window by configuring a View.

====

In looking into [Chrstia's question](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14604#issuecomment-1670110016) it became clear I didn't understand this ticket as I thought I did during refinement. Hopefully the below provides some clarity.

In #13220 I wrote an Entity Browser plugin that causes the Q&A table links to open in a new tab when clicked. Below shows the links in that were affected by that plugin. Note that this is what you would see _after_ selecting some Q&As inside the modal.

![Screenshot 2023-08-08 at 3 04 41 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/221539/5f694a07-7c75-4329-a29d-80b296e55f5b)

This ticket, however, is referring to the links _inside the modal_, which have as I understand it, always opened inside the modal. The modal links are provided by Entity Browser, but the plugin I wrote for #13220 doesn't apply to these links. Instead, the list of content populating the modal is coming from a [View](https://va-gov-cms.ddev.site/admin/structure/views/view/content_entity_browsers). As far as I can tell, there have been no modifications to the View since Aug 2022.

Since the modal content is controlled by a View, we have lots of options for forcing opening in a new window/tab.

**Comment by chri5tia** [comment]
This makes perfect sense Daniel, and the work that was done was just as important. Thanks for the sanity check. [This PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/14723) contains the view config change for link in the modal.
